### PR TITLE
feat: handle optional data #7

### PR DESCRIPTION
**Pull request by nabdali**

### CODE REVIEW

File: src/main/java/com/ozwillo/socatelgraphql/repository/PostRepository.java [hunk 1]
```diff
@@ -1,186 +1 @@
...
        return Queries.SELECT()
                .prefix(SOCATEL, RDF, SIOC, GN, FOAF)
                .select(var("post"), var("identifier"), var("description"),
                        var("creationDate"), var("language"), var("num_likes"),
                        var("num_replies"), var("location_name"), var("location_alternateName"),
                        var("location_countryCode"), var("owner_identifier"), var("owner_title"),
                        var("owner_description"), var("owner_webLink"), var("owner_language"),
                        var("owner_num_likes"), var("creator_name"), var("creator_username"));
    }
}
```
Comment (bobeal) [inline src/main/java/com/ozwillo/socatelgraphql/repository/PostRepository.java]: 😨

**Comment by bobeal** [comment]
I'm not really convinced with the list of fields identified as optional (eg a tweet always has a creator, no ?)

File: src/main/java/com/ozwillo/socatelgraphql/repository/PostRepository.java (hunk 1 above)
Comment (nabdali) [reply src/main/java/com/ozwillo/socatelgraphql/repository/PostRepository.java]: what is this ^^

Comment (nabdali) [reply src/main/java/com/ozwillo/socatelgraphql/repository/PostRepository.java]: I think is my git client, because on my editor I don't have same thing

**Comment by nabdali** [comment]
> I'm not really convinced with the list of fields identified as optional (eg a tweet always has a creator, no ?)

Yes in reality only `location` is optional

**Comment by nabdali** [comment]
> > I'm not really convinced with the list of fields identified as optional (eg a tweet always has a creator, no ?)
> 
> Yes in reality only `location` is optional

but many post don't have a owner (like this [post](https://socatel-ts.ozwillo-preprod.eu/resource?uri=http:%2F%2Fwww.everis.es%2FSoCaTel%2Fontology%23Post_1101909141730275328) )

**Pull request merged**